### PR TITLE
Update feast doc, fix issue

### PR DESCRIPTION
**Pull request by AmyHoney**

### CODE REVIEW

**Comment by AmyHoney** [comment]
Fix issue: TypeError: cannot create weak reference to 'property' object

File: docs/user-guide/feast.rst [hunk 1]
```diff
@@ -74,24 +100,11 @@ We've made some dummy data for this workshop in `infra/driver_stats.parquet <htt
...
-    $ microk8s kubectl get secret -n admin | grep minio
-
-    # get the access key for MinIO
-    $ microk8s kubectl get secret <minio-secret-name> -n admin -o jsonpath='{.data.accesskey}' | base64 -d
-
-    # get the secret key for MinIO
-    $ microk8s kubectl get secret <minio-secret-name> -n admin -o jsonpath='{.data.secretkey}' | base64 -d
-
-Secondly update the MinIO parameters on your environment, create a bucket for feast and upload data files to bucket.
+You need to use your MinIO parameters obtained befor to update the MinIO parameters, create a bucket for feast and upload data files to bucket.
```
Comment (skolavmware) [inline docs/user-guide/feast.rst:107]: Small typo - before

File: docs/user-guide/feast.rst [hunk 2]
```diff
@@ -49,20 +69,26 @@ Before we get started, first install some dependencies and Feast:
...
-    pip install boto3
-    pip install s3fs
-    pip install feast==0.29.0
+    !pip install scikit-learn
+    !pip install "numpy>=1.16.5,<1.23.0"
+    !pip install pyarrow
+    !pip install fastparquet
+    !pip install boto3
+    !pip install s3fs
+    !pip install feast==0.29.0
```
Comment (skolavmware) [inline docs/user-guide/feast.rst:79]: We can also add - apt-get install wget,  as we are using wget in below lines

File: docs/user-guide/feast.rst [hunk 3]
```diff
@@ -128,22 +141,13 @@ Secondly update the MinIO parameters on your environment, create a bucket for fe
...
-
-    feast init <your_feast_project_name>
-  
-
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-Use your configured bucket
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Write **data_sources.py** file to load data from S3 storage
+New a python file to write **data_sources.py** file to load data from S3 storage
```
Comment (skolavmware) [inline docs/user-guide/feast.rst:150]: Its always better to tell where to copy these python files, is it on kubeflow notebook or terminal

**Review by skolavmware — CHANGES_REQUESTED** [review]
Small changes requested

File: docs/user-guide/feast.rst (hunk 2 above)
Comment (AmyHoney) [reply docs/user-guide/feast.rst:79]: Thanks Srinivas. `wget` has installed in these kubeflow Docker Images. Thus, there is no necessary to install wget manually.

Comment (skolavmware) [reply docs/user-guide/feast.rst:79]: right but when we testing this on the nimbus VM we need to install it explicitly

**Review by liuqi — APPROVED** [review]

**Pull request merged**